wrappers/services_test.go

```diff
@@ -3180,6 +3221,60 @@ apps:
 	})
 }
 
+func (s *servicesTestSuite) TestRestartInDifferentStates(c *C) {
+	const manyServicesYaml = `name: test-snap
+version: 1.0
+apps:
+  svc1:
+    command: bin/foo
+    daemon: simple
+  svc2:
+    command: bin/foo
+    daemon: simple
+  svc3:
+    command: bin/foo
+    daemon: simple
+  svc4:
+    command: bin/foo
+    daemon: simple
+`
+	srvFile1 := "snap.test-snap.svc1.service"
+	srvFile2 := "snap.test-snap.svc2.service"
+	srvFile3 := "snap.test-snap.svc3.service"
+	srvFile4 := "snap.test-snap.svc4.service"
+
+	info := snaptest.MockSnap(c, manyServicesYaml, &snap.SideInfo{Revision: snap.R(1)})
+
+	r := systemd.MockSystemctl(func(cmd ...string) ([]byte, error) {
+		states := map[string]serviceState{
+			srvFile1: serviceState{"active", "enabled"},
+			srvFile2: serviceState{"inactive", "enabled"},
+			srvFile3: serviceState{"active", "disabled"},
+			srvFile4: serviceState{"inactive", "disabled"},
```

Comment:
`srvFile1: serviceState{...` can be just `srvFile1: { ...` (repeating serviceState is redundant and I expect unit tests to fail on that).